examples/QuickStart/README.md

```diff
@@ -0,0 +1,26 @@
+## Quick Start Example
```

Comment:
I think users may want to know whether their program is running in a correct way,
How do they check if the program result is right or wrong?

I suggest Adding the expected performance(e.g episode_reward)  after the convergence of training